integration/entropy/entropy_test.bats

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source /etc/os-release || source /usr/lib/os-release
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+# Environment variables
+IMAGE="busybox"
+MIN_ENTROPY_LEVEL="1000"
```

Comment:
We should probably have a comment stating why 1000.
We should probably also have a top level comment describing what this test is testing, and why.

One thing - with the kernels larger than, what was it, 4.14.36 maybe, won't the VM kernel wait (block) until it has enough entropy, and then start - in which case, if we are testing inside the container, what exactly are we testing for and expecting? Hence, the request for clarification above with the addition of some comments ;-)
/cc @jcvenegas